### DISCUSSION TITLE
use of encryption sdk is superfluous and burdensome

### DISCUSSION THREAD

**Discussion by kapilt** [aws-cloudformation/cloudformation-cli-python-plugin#197] [Ideas]
afaics the use for the hook library usage on the encryption sdk is doing fairly trivial stuff that would be easily replaced by a call to kms_client.decrypt .. the use of the encryption sdk brings in native compiled extensions requirements and dependencies (cryptography library), and in general is targeted towards more advanced usage then what is needed here.  removing it in favor of the simple thing seems like an easy usability/deployment win.

**Comment by benkehoe** [comment]
Given that the library is only doing decryption, wouldn't this need to be a change across all language SDKs?

**Comment by kapilt** [comment]
its not clear why this would need any change to the language sdks, this is simply removing the superfluous use of aws_encryption_sdk for a boto call to kms.decrypt

**Comment by kapilt** [comment]
this was addressed in https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/pull/260
